nemo_skills/evaluation/evaluator.py

```diff
@@ -262,11 +262,35 @@ def eval_arena(cfg):
             Path(output_file).unlink()
 
 
+@nested_dataclass(kw_only=True)
+class LeanEvaluatorConfig:
+    sandbox: dict = field(default_factory=lambda: {'sandbox_type': 'local'})
+    num_parallel_requests: int = 10
+    in_memory_lines: int = 500
+    timeout: float = 30.0
+    ignore_cache: bool = False
+
+
+def eval_lean4(cfg):
+    eval_config = LeanEvaluatorConfig(**cfg.eval_config)
+    from nemo_skills.code_execution.sandbox import get_sandbox
```

Comment:
Can you put this on top (and same for other places where it's imported inside)? We don't really need to hide this import anymore

Reply:
Done. Do you want me to also put  `from nemo_skills.inference.server.model import get_model` up? For example in setup of metrics?